src/style.css

```diff
@@ -1,3 +1,66 @@
-body {
+body,
+.nav {
   margin: 0;
+  padding: 0;
+  max-width: 1200px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  height: 60px;
+  padding-right: 50px;
+  padding-left: 50px;
+}
+
+.nav {
+  justify-content: space-between;
+  display: flex;
```

Comment:
There is no need to use justify-content: space-between for the .nav class, as it is already set in the .header class.

Reply:
![man-mechanic_1f468-200d-1f527](https://github.com/mate-academy/layout_moyo-header/assets/129754902/27bd7a7f-8c9d-4cbe-9e13-845928ce7e4b)